### DISCUSSION TITLE
React+PocketBase oAuth2.0 Demo

### DISCUSSION THREAD

**Discussion by rajesh6161** [pocketbase/pocketbase#820] [Show and tell]
I have created a demo for oAuth2.0 in React Apps using PockeBase with proper state management using Redux and latest react-router-dom for routing. Please have a look:
https://github.com/rajesh6161/pocketbase-oauth-demo

![image](https://user-images.githubusercontent.com/40054161/197317563-cb4eaa64-894d-4f57-809b-bb9e66858d72.png)

![image](https://user-images.githubusercontent.com/40054161/197317573-a902cd7f-8a93-499a-a811-4713c78dca6b.png)

**Comment by benallfree** [comment]
You did it again! Nice job. Added to https://github.com/benallfree/awesome-pocketbase in the Demo section.
